### PR TITLE
DATAUP-494: style housekeeping and reworking job params display

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Two main tasks:
- ensure that the job param display removes system variables
- tidy up the margins/padding in the bulk import cell to fix some small layout problems

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-494
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to see the exciting new borders/margins on the tabs of the bulk import cell

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/scss/modules/_colours.scss [hunk 1]
```diff
@@ -119,6 +119,9 @@ $kbase-palette: (
   "disabled-light": rgb(242, 239, 235),
   "disabled": rgb(222, 213, 203),
   "disabled-dark": rgb(157, 147, 137),
+
+  // other misc colours
+  "mid-blue": rgb(33, 150, 243),
```
Comment (ialarmedalien) [inline kbase-extension/scss/modules/_colours.scss:124]: tokenise this colour throughout the SCSS files

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 2]
```diff
@@ -120,8 +117,8 @@ $tab-widget-padding: 4px;
         // .kb-bulk-import-configure__filetype_panel__filetype_icon
         &_icon {
           padding: 2px 8px 0;
-          width: 3rem;
           text-align: center;
+          width: 3rem;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_bulkImportCell.scss:121]: alphabetise

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ define(['bluebird', 'common/html', './jobStatusTable'], (Promise, html, JobStatu
 
         function renderLayout() {
             return div({
-                class: 'kb-job__container',
+                class: 'kb-job-status-tab__container',
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js:13]: standardise name

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 4]
```diff
@@ -92,22 +91,31 @@ define([
...
-                            return param === p.id;
-                        })
-                        .map((p) => {
-                            return p.ui_name;
-                        })[0];
-                    params[param] = {
-                        value,
-                        label,
-                    };
+                    // find each parameter in job_params in the app specs
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:100]: The other part of this ticket: ensure that when the job params are displayed, only the user params are shown. We don't need to show any of the private/system variables.

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 5]
```diff
@@ -284,7 +292,7 @@ define([
                             },
                         },
                     ],
-                    fnDrawCallback: () => {
+                    drawCallback: () => {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:295]: `fnDrawCallback` is the legacy name

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/results/resultsTab.js [hunk 6]
```diff
@@ -163,13 +163,13 @@ define(['bluebird', 'common/ui', 'common/events', './outputWidget', './reportWid
 
             const objectNode = document.createElement('div');
             const reportNode = document.createElement('div');
+            const containerNode = document.createElement('div');
+            containerNode.classList.add('hidden', 'kb-result-tab__container');
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/results/resultsTab.js:167]: Rejig slightly to add a container around the tab contents. This simplifies the styling slightly.

File: test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js [hunk 7]
```diff
@@ -138,7 +145,12 @@ define([
         paramTests = [
             {
                 input: {
-                    job_params: [{ name: 'some_old_file' }],
+                    job_params: [
+                        {
+                            name: 'some_old_file',
+                            workspace_id: 'some_workspace_id',
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js:151]: make sure these `workspace_id` values don't show up when params are generated by the job status table

File: kbase-extension/scss/modules/_mixins.scss [hunk 8]
```diff
@@ -30,3 +30,7 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+@mixin default-container-padding {
+  padding: $padding-large-horizontal;
```
Comment (ialarmedalien) [inline kbase-extension/scss/modules/_mixins.scss:35]: shortcut for tokenising the container padding

**Review by briehl — APPROVED** [review]
🎉

**Pull request merged**